test/inventoryDb/fixtures.js

```diff
@@ -0,0 +1,55 @@
+const AVAILABILITY_YEAR_MONTH = '2018-03';
+const SINGLE_AVAILABILITY_DAY = '26';
```

Comment:
Are these supposed to be strings?

Reply:
Yes, these are intended for easy construction of date strings. Having said that, the date values will be updated to numbers in an upcoming pull request due to the need to do math with them.